pkg/csi/service/common/constants.go

```diff
@@ -162,17 +162,11 @@ const (
 	// IPs is Client IP address, IP range or IP subnet
 	IPs string = "ips"
 
-	// CSINamespaceVanillaK8S is the namespace of CSI driver in Vanilla K8S
-	CSINamespaceVanillaK8S = "kube-system"
+	// CNSCSINamespace is the namespace of CSI in Supervisor Cluster
+	CNSCSINamespace = "vmware-system-csi"
```

Comment:
Where `CNSCSINamespace` is this used in this PR?

Reply:
Renamed and used it in the non Vanilla k8s related code